src/pulse.cpp

```diff
@@ -29,7 +29,7 @@ int Pulse::init(SubscriptionType subscriptionType,
 void Pulse::sink_input_info_callback(pa_context *, const pa_sink_input_info *i,
                                      int, void *userdata) {
   Data *data = (Data *)userdata;
-  if (i && !i->corked) data->activeSink = true;
+if (i && !i->corked && i->client != 0) data->activeSink = true;
```

Comment:
Fix indentation and replace the 0 with `PA_INVALID_INDEX`

Reply:
Should also be checked in the `source_output_info_callback` function too :)

Reply:
Thank you for giving me a way to test the issue with it crashing. Experimented a bit but not found a way yet that prevents the crash and still functions in inhibiting.

Might tinker some more if I find the time in between my regular work schedule. Certainly an interesting problem to look into further.

Reply:
I'll merge this in the mean time :)